calendar.html

```diff
@@ -15,10 +15,21 @@
 ---
 
 <div class="calendar-controls">
-      <button class="calendar-button hideable" id="css-calendar-switch">Switch View</button>
-      <a class="calendar-button" href="{{ page.calendar_url }}">Add to own Calendar</a>
+  <button class="calendar-button hideable" id="css-calendar-switch">Switch View</button>
+  <a class="calendar-button" href="{{ page.calendar_url }}">Add to own Calendar</a>
 </div>
 
 <div class="calendar-container">
+  <div class="calendar-event" id="calendar-event">
+      
+      <h2 class="event-tag event-tag-title">Event:</h2> <span class="event-text" id="event-text-title"></span><br>
+      <h2 class="event-tag event-tag-date">Date:</h2> <span class="event-text" id="event-text-date"></span><br>
```

Comment:
We can put the span on the next line - breaks in the html don't make a difference to the actual rendered content.

Reply:
Having `h2 span br` on each line made sense to look like it is displayed. Would you rather have 
```
h2
span
br
```
on separate lines?

Reply:
Yeah, I see what you mean... Idea: maybe place the h2 and span inside a <div>, then they should all appear by themselves? If that doesn't work, then yeah, I would go with separate lines.

Reply:
Enclosed in `pre`, kept formatting for each `h2 span` in one line.